test/index.ts

```diff
@@ -39,3 +55,51 @@ describe('Queries Test', () => {
     expect(queryResult).to.be.deep.eq({ data: { hello: 'Hello World!' } });
   });
 });
+
+describe('Mutation Test', () => {
+  it('Create Mutation', async () => {
+    const createUseMutation = await axios({
+      url,
+      method: 'post',
+      data: {
+        query: `
+          mutation CreateUser($credentials: UserInput!) {
+            createUser(data: $credentials) {
+              id,
+              name,
+              email,
+              birthDate
+            }
+          }
+        `,
+        variables: { credentials: testUser },
+      },
+    });
+
+    expect(createUseMutation.data.data.createUser.email).to.be.eq(testUser.email);
+    expect(createUseMutation.data.data.createUser.name).to.be.eq(testUser.name);
+    expect(createUseMutation.data.data.createUser.birthDate).to.be.eq(testUser.birthDate);
+    expect(createUseMutation.data.data.createUser.id).to.exist;
```

Comment:
Pra uma checagem de objeto mais completa, tendo em vista os retornos do GraphQL, eu costumo sugerir o `to.be.deep.eq`, porque ele faz uma comparação estrita de objetos, ou seja, confere todos as chaves e valores e falha se tiver campo a mais ou a menos.

Reply:
é que o resultado da mutation contem o ID e o testUser possui o PASSWORD. Dai teria que mudar os dois, não?